python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py

```diff
@@ -231,7 +230,7 @@ def _get_in_progress_runs_for_assets(
         run = record.pipeline_run
         asset_selection = run.asset_selection
         run_step_keys = graphene_info.context.instance.get_execution_plan_snapshot(
-            run.execution_plan_snapshot_id
+            check.not_none(run.execution_plan_snapshot_id)
```

Comment:
Scary at first but looks like this codepath would break anyways if this were None.

Reply:
exactly

Reply:
In the future it would be very helpful to somehow note that in either a comment or in the text of the check error so that it isn't left as an exercise to the reader if this introduce new errors or trigger them earlier.